Makefile

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Protect against invocation without a specific target.
-all:
+all: build
```

Comment:
Is `all` a standard target for the CI?

Reply:
My understanding is that `prow` is requiring that https://github.com/kubernetes-csi/external-snapshotter/blob/master/release-tools/prow.sh#L1247

Reply:
Hmm... that indeed appears to be the case. So I withdraw my objection!